.eslintrc.json

```diff
@@ -1,6 +1,11 @@
 {
   "plugins": ["github", "jest", "@typescript-eslint"],
```

Comment:
To work around the build failures caused by [this bug](https://github.com/github/eslint-plugin-github/pull/326) in eslint-plugin-github v4.4.0, we'll need to add `eslint-plugin-escompat` as a dependency and include it in the list of plugins here (sigh).